forum_leaderboard/static/js/src/forum_leaderboard_studio.js

```diff
@@ -0,0 +1,21 @@
+/* Javascript for ForumLeaderboardXBlock. */
+function ForumLeaderboardStudioXBlock(runtime, element) {
+    $('.save-button', element).bind('click', function() {
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+        var data = {
+            discussion_id: $(element).find('input[name=discussion_id]').val(),
+            count: $(element).find('input[name=count]').val()
+        };
+        $.post(handlerUrl, JSON.stringify(data)).success(function(response) {
+            if (response['success']) {
+                window.location.reload(false);
+                return
+            }
+            alert(response['errors'].join('\n'));
```

Comment:
@bradenmacdonald I imagine it would be better to leave things as they are and let you iterate on top of them, since then you'll have less conflicts to resolve and you can make consolidated code on the fly without me incurring redundant work.

@antoviaque Ready to merge when you are, by the look of it.